### ISSUE TITLE
ReplyKeyboardMarkup rows parameter

### ISSUE THREAD

**Issue by Goodjee** [ayrat555/frankenstein#89]
I've could not find a way to make custom keyboard shema.  
row parameter is missing in  replyKeyboardMarkup schema.

**Comment by ayrat555** [comment]
Hello. can you please take a look at this example https://github.com/ayrat555/frankenstein/blob/master/examples/reply_keyboard.rs

**Comment by Goodjee** [comment]
I did! But unfortunately i did not find how to markup keybord, for example like on this picture:
![InlineKeyboardMarkup-full](https://user-images.githubusercontent.com/50458483/190888080-aefd536f-9573-41d5-bd11-a75e01d0ba7a.jpg)

I think some functionality from telegram API is missing.

**Comment by ayrat555** [comment]
good to hear that you have some progress. 

what do you mean by "how to markup keyboard," ?

**Comment by Goodjee** [comment]
Thanks for your time. I've figured it out. I needed to make keyboard with 2 buttons on first line and 1 button on second line.
I didn't see that keyboard is a Vector of keyboard vectors:
`let mut keyboard: Vec<Vec<KeyboardButton>> = Vec::new();
 let mut row: Vec<KeyboardButton> = Vec::new();
 let mut row2: Vec<KeyboardButton> = Vec::new();
 row.push(button);
 row.push(button2);
 row2.push(button3);
 keyboard.push(row);
 keyboard.push(row2);`
